src/data/pi/lex.lean

```diff
@@ -60,44 +76,38 @@ instance lex.is_strict_order [linear_order ι] [∀ a, partial_order (β a)] :
         ⟨N₂, λ j hj, (lt_N₁ _ (hj.trans H)).trans (lt_N₂ _ hj), (lt_N₁ _ H).symm ▸ b_lt_c⟩]
     end }
 
-instance [linear_order ι] [Π a, partial_order (β a)] : partial_order (lex (Π i, β i)) :=
+instance [linear_order ι] [Π a, partial_order (β a)] : partial_order (Πₗ i, β i) :=
 partial_order_of_SO (<)
 
-protected lemma lex.is_strict_total_order' [linear_order ι]
-  (wf : well_founded ((<) : ι → ι → Prop)) [∀ a, linear_order (β a)] :
-  is_strict_total_order' (lex (Π i, β i)) (<) :=
-{ trichotomous := λ a b,
-    begin
-      by_cases h : ∃ i, a i ≠ b i,
-      { let i := wf.min _ h,
-        have hlt_i : ∀ j < i, a j = b j,
-        { intro j, rw ← not_imp_not,
-          exact λ h', wf.not_lt_min _ _ h' },
-        have : a i ≠ b i, from wf.min_mem _ h,
-        exact this.lt_or_lt.imp (λ h, ⟨i, hlt_i, h⟩)
-          (λ h, or.inr ⟨i, λ j hj, (hlt_i j hj).symm, h⟩) },
-      { push_neg at h, exact or.inr (or.inl (funext h)) }
-    end }
-
 /-- `Πₗ i, α i` is a linear order if the original order is well-founded.
 This cannot be an instance, since it depends on the well-foundedness of `<`. -/
```

Comment:
```suggestion
/-- `Πₗ i, α i` is a linear order if the original order is well-founded.
This cannot be an instance, since it depends on the well-foundedness of `<`. -/
```